src/core/Microsoft.PowerToys.Settings.UI/Views/PowerRenamePage.xaml

```diff
@@ -122,7 +122,9 @@
                     HorizontalAlignment="Left"
                     Margin="{StaticResource SmallTopBottomMargin}"
                     RelativePanel.Below="DescriptionPanel">
+                <HyperlinkButton NavigateUri="https://aka.ms/PowerToysOverview_PowerRename">
```

Comment:
These links should point to line module overview link to reduce complexity.

Reply:
Store the URLs as a XAML Resource?